demo-app/src/app/ot/playground/page.tsx

```diff
@@ -1,17 +1,14 @@
-import { parseYaml, applyConfiguration } from '@open-truss/open-truss'
-import * as COMPONENTS from '@open-truss-components'
-const configurationFunction = applyConfiguration(COMPONENTS)
-
 async function PlaygroundPage(): Promise<JSX.Element> {
-  const parsedConfig = parseYaml(config)
-  const renderedComponents = configurationFunction(parsedConfig, {})
-
-  return <>{renderedComponents}</>
+  return <div style={{ whiteSpace: 'pre' }}>{config}</div>
 }
 
 export default PlaygroundPage
 
 const config = `
+To use the playground you can create a new Open Truss configuration at src/open-truss/configs/1-your-config.yaml. For example:
+
+src/open-truss/configs/1-foo-bar.yaml
```

Comment:
Hmmm. Perhaps we should put playground configs in their own directory as distinct from normal configs?

Maybe `src/open-truss/configs/playground/1-foo-bar.yaml`?